mkdocs.yml

```diff
@@ -67,7 +67,7 @@ nav:
                 - Installation: distributions/gentoo/installation.md
           - Manjaro:
                 - Home: distributions/manjaro/home.md
-                - Building: distributions/manjaro/building.md
+                - Building (Outdated, deprecated): distributions/manjaro/building.md
```

Comment:
I think it's fine to remove this one as I don't think anything links to it.

Alternatively, you could put instructions to build the new kernel and iso on this page instead of the older instructions.